shared/actions/chat2/index.tsx

```diff
@@ -3387,6 +3287,7 @@ function* chat2Saga() {
     'onGetInboxUnverifiedConvs'
   )
   yield* Saga.chainAction2(EngineGen.chat1ChatUiChatInboxFailed, onGetInboxConvFailed, 'onGetInboxConvFailed')
+  yield* Saga.chainAction2(EngineGen.chat1ChatUiChatInboxLayout, onChatInboxLayout, 'onChatInboxLayout')
```

Comment:
we want to move towards having it be `action to response` with the chains, so `onChatInboxLayout` doesn't really match that. ideally it'd be something like `shouldChangeSelectedConversation` or something